releasenotes.txt

```diff
@@ -1,4 +1,185 @@
-﻿Build 5.3.14
+﻿Build 5.4.0
+=============================
+
+Release notes - NHibernate - Version 5.4.0
+
+** Highlights
+    * A new batching strategy is available, minimizing the batching memory footprint. See #2959. Using it may increase CPU usage.
+    * 199 issues were resolved in this release.
```

Comment:
Anything else worth highlighting?

Or even just go for:

```suggestion
199 issues were resolved in this release.
```

Reply:
New target frameworks: .NET Framework 4.8,  .NET Standard 2.1 and .NET 6